insights/client/__init__.py

```diff
@@ -268,7 +267,7 @@ def update(self):
         else:
             logger.debug("Egg update disabled")
 
-    def verify(self, egg_path, gpg_key=constants.pub_gpg_path):
+    def verify(self, egg_path):
```

Comment:
The most correct approach would then be:

```suggestion
    def verify(self, egg_path, gpg_key):
```

But that would require moving the default value to the calling module, increasing the amount of changes. And since I believe we **should** aim for the smallest possible amount, I’d be most in favor of keeping the original code:

```suggestion
    def verify(self, egg_path, gpg_key=constants.pub_gpg_path):
```

This would increase an inconsistency with the _validate_gpg_sig_ function that doesn’t take a key argument, and references _constants.pub_gpg_path_ in its body. I am slighlty inclined towards not changing the signature, as it would feel more as a refactoring in order to increase consistency.

I can, however, live with the argument being moved out. Our goal is to have the code merged, not to endlessly discuss every tiny bit.